app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -363,7 +369,7 @@ function FileInput(props: FileInputProps, ref: ForwardedRef<any>) {
           {!value && !isValuePending && (
             <div className="usa-file-input__instructions" aria-hidden="true">
               {bannerText && <strong className="usa-file-input__banner-text">{bannerText}</strong>}
-              {isMobile && bannerText ? null : (
+              {!showInnerHint ? null : (
```

Comment:
I think as long as we can guarantee that this is a boolean, we could turn this positive and do syntax like...

```suggestion
              {showInnerHint && (
```

Seems more readable that way.

The ternary may have been necessary since React does funny things when you give it "falsey" values like an empty string or the number `0`, other than just not rendering anything.

Example: https://codepen.io/aduth/pen/dWwzrL